### DISCUSSION TITLE
Is there an 8bit library for jax similar to bitsandbytes?

### DISCUSSION THREAD

**Discussion by aniquetahir** [jax-ml/jax#17256] [Q&A]
BitsandBytes library contains optimizers, multiplication and quantization functions. Is there an equivalent for jax/optax?
https://github.com/TimDettmers/bitsandbytes

**Comment by davidearlyoung** [comment]
Having a 8-bit quant of a model would be interesting to see how well it does. And to make it more accessible and cost less with resources and time to use.
